### DISCUSSION TITLE
Code-server docker and Forward a Port by vs code

### DISCUSSION THREAD

**Discussion by anduranm** [coder/code-server#6944] [General]
Hi friends, I have a code-server instance in docker, the problem comes when I try to forward a port in visual studio code

![image](https://github.com/user-attachments/assets/3f9f9993-ddcc-473c-9f63-a074702544a8)

it works on port 8888
![image](https://github.com/user-attachments/assets/432eb071-8e53-4118-adeb-8be476613de5)

but if I tried to use the link by tunnel

that's what happened 
![image](https://github.com/user-attachments/assets/fc569846-281f-4067-b2da-6cf05b4e6e26)

![image](https://github.com/user-attachments/assets/e51f7950-1093-4563-8369-1e5fb3c6b3cb)

is there a solution?

**Comment by code-asher** [comment]
devtunnels.ms is a Microsoft thing, right?  There might not be anything we can do about it.  It looks like maybe it is not forwarding the web sockets but I am not sure.  The only thing I could find was https://github.com/microsoft/dev-tunnels/issues/291
